test/vault_helpers.go

```diff
@@ -24,6 +24,7 @@ import (
 )
 
 const REPO_ROOT = "../"
+const AUTO_UNSEAL_KMS_KEY_ALIAS = "dedicated-test-key"
```

Comment:
Add a comment explaining what this is, both so we know what this is, and open source community folks know to update this.